src/routes/replies.py

```diff
@@ -0,0 +1,55 @@
+import logging
+from datetime import datetime
+
+from flask import Blueprint
+from flask_jwt_extended import get_jwt_identity, jwt_required
+from flask_restful import (Api, Resource, abort, fields, inputs, marshal_with, reqparse)
+from sqlalchemy import or_
+from typing import Optional
+from src.new_backend.models import Collection, Comment, Paper, Reply, db, User
+from .user_utils import get_user
+
+app = Blueprint('replies', __name__)
+api = Api(app)
+logger = logging.getLogger(__name__)
+
+replies_fields = {
+    'id': fields.String,
+    'user': fields.String(attribute='user.username'),
+    'text': fields.String,
+    'createdAt': fields.DateTime(dt_format='rfc822', attribute='creation_date'),
+}
+
+EMPTY_FIELD_MSG = 'This field cannot be blank'
+
+class ReplyResource(Resource):
```

Comment:
It might be confusing to have 2 `ReplyResource` in the code base. I suggest consolidating the two resources and replacing the `post` method below with `patch`

Reply:
It'll also ensure that we won't have two copies of `replies_fields`

Reply:
I agree with you. I would suggest the owner of the code to clean and remove unused code because it can be confusing. For example, MongoDB seems to be replaced with PostgreSQL. Also, new_backend, main, app, src in the same project makes it a little bit confusing. I think one common convention is app/main.py.

Reply:
You're absolutely right (and I apologize about that), the code is still patchy but we're working on fixing that.  
Would you like to update your code and we'll take with re-organizing the rest?